cfgov/cfgov/settings/local.py

```diff
@@ -45,3 +45,12 @@
 
 
 MIDDLEWARE_CLASSES += CSP_MIDDLEWARE_CLASSES
+
+# Define caches necessary for eRegs.
+# Disable caching when working locally.
+CACHES = {
+    k: {
+        'BACKEND': 'django.core.cache.backends.dummy.DummyCache',
+        'TIMEOUT': 0,
+    } for k in ('default', 'eregs_longterm_cache', 'api_cache')
+}
```

Comment:
With this in `settings/local.py` and not `settings/base.py`, will this break running the regulations-site and regulations-core unittests with `tox -e optional`?

Reply:
I thought of that, but `cfgov.settings.test` imports from `cfgov.settings.local`, so we should be okay. More generally, I thought that we were deprecating pursuit of the optional `tox` configs?

Reply:
Oh, good points!